Document query tuning with relationship indexes (#1099)
neo4j/neo4j-documentation: cypher/cypher-docs/src/docs/dev/query-tuning-indexes.adoc

Therefore, it is important to have a fundamental understanding of how the indexes operate.
This section describes the query plans that result from different index scenarios.

Node indexes and relationship indexes operate in the same way.

> Therefore, we will use node and relationship indexes interchangeably in this section.

Reviewer: ```suggestion
Therefore, node and relationship indexes are used interchangeably in this section.
```
